test/utils/MockConnection.js

```diff
@@ -23,6 +27,8 @@ class MockConnection extends Connection {
     process.nextTick(() => {
       if (!aborted) {
         callback(null, stream)
+      } else {
+        callback(new RequestAbortedError(), null)
```

Comment:
Tomayto, tomahto :)
You can read it in both ways, if the operation has not been aborted, then continue, otherwise return a RequestAbortedError error.

Reply:
Read it, I can.  Merge it, we can:)